src/main/java/org/prebid/server/hooks/execution/model/GroupExecutionOutcome.java

```diff
@@ -0,0 +1,11 @@
+package org.prebid.server.hooks.execution.model;
+
+import lombok.Value;
+
+import java.util.List;
+
+@Value(staticConstructor = "of")
+public class GroupExecutionOutcome {
+
+    List<HookExecutionOutcome> hooks;
```

Comment:
Should it be like `hookOutcomes`?
Also, if yes - pls check StageExecutionOutcome.groups.

Reply:
I picked short name on purpose as `executionOutcome` or `outcome` suffix doesn't add any useful bits. It's already in Stage or GroupExecutionContext so there is no reason to repeat the suffix in my opinion. Next PR's exploring usage of these models may prove this assumption wrong of course so it makes sense to defer final decision for later.